appveyor.yml

```diff
@@ -0,0 +1,43 @@
+environment:
+  P: "c:/projects/libs"
+
+# Operating system (build VM template)
+os: Visual Studio 2015
+
+# scripts that are called at very beginning, before repo cloning
+init:
+  - git config --global core.autocrlf input
+
+
+# clone directory
+clone_folder: c:/projects/openmeeg
+
+platform: x64
+configuration: Release
+
+install:
+  # by default, all script lines are interpreted as batch
+
+build:
+  project: INSTALL.vcxproj      # path to Visual Studio solution or project
+
+# scripts to run before build
+before_build:
+  - echo Running cmake...
+  - cd c:/projects/openmeeg
+  - cmake -G "Visual Studio 14 2015 Win64" -DCMAKE_INSTALL_PREFIX=%P% -DBUILD_SHARED=ON -DBUILD_DOCUMENTATION=OFF -DBUILD_TESTING:BOOL=ON -DENABLE_PYTHON=OFF -DENABLE_PACKAGING:BOOL=OFF -DUSE_VTK=OFF -DUSE_ATLAS=OFF -DUSE_SYSTEM_matio:BOOL=OFF -DUSE_SYSTEM_hdf5:BOOL=OFF -DUSE_SYSTEM_zlib=OFF -DCMAKE_SKIP_RPATH:BOOL=OFF -DUSE_LAPACK=ON .
+  - cmake --build . --config Release
+
+# scripts to run after build
+after_build:
+  - cd %P%
+  - 7z a c:/projects/openmeeg/openmeeg.zip *.exe -tzip
+  - cd c:/projects/openmeeg/OpenMEEG/build
+  - ctest -V
+
+artifacts:
+  - path: openmeeg.zip
+    name: openmeeg.zip
```

Comment:
does it properly generate the artifacts?

I suspect you need to call cpack or make package to generate the windows installers.

what would be awesome would be to update the builts from master branch to a download website. Don't know how to do this...

we would probably need an ssh key to upload somewhere...